qiime2/metadata/metadata.py

```diff
@@ -203,6 +203,54 @@ def _filter_ids_helper(cls, df_or_series, ids, ids_to_keep):
         return df_or_series.drop(labels=ids_to_discard, axis='index',
                                  inplace=False, errors='raise')
 
+    def save(self, filepath, ext=None):
+        """Save a TSV metadata file.
+
+        The TSV metadata file format is described at https://docs.qiime2.org in
+        the Metadata Tutorial.
+
+        The file will always include the ``#q2:types`` directive in order to
+        make the file roundtrippable without relying on column type inference.
+
+        Parameters
+        ----------
+        filepath : str
+            Path to save TSV metadata file at.
+
+        ext : str
+            Preferred file extension (.tsv, .txt, etc).
+            Will be left blank if no extension is included.
+            Including a period in the extension is
+            optional, and any additional periods delimiting
+            the filepath and the extension will be reduced
+            to a single period.
+
+        Returns
+        -------
+        str
+            Filepath and extension (if provided) that the
+            file was saved to.
+
+        See Also
+        --------
+        load
```

Comment:
Now that this docstring is shared via this parent class, you'll need to update this to let all the children know _which_ `load` method to use:

```suggestion
        Metadata.load
```